guides/common/assembly_using-external-databases.adoc

```diff
@@ -15,7 +15,7 @@ endif::[]
 To create and use external databases for {Project}, you must complete the following procedures:
 
 . xref:preparing-a-host-for-external-databases_{context}[].
-Prepare a {RHEL} 8 server to host the external databases.
+Prepare a {EL} 9 or {EL} 8 server to host the external databases.
```

Comment:
https://github.com/theforeman/foreman-documentation/pull/2396#discussion_r1308203116

Reply:
Shrinked to `{EL}  server` :+1: